doc/Command-Reference.md

```diff
@@ -1863,7 +1969,7 @@ This sub-section explains the various IP protocol specific show commands that ar
 This command displays either all the route entries from the routing table or a specific route.
 
   - Usage:  
-    show ip route [<ip-address>]`
+    show ip route <ip_address>

```

Comment:
Why remove the square brackets? Is <ip_address> no longer optional?

Reply:
It was removed by mistake. There was an issue in using '<' within square brackets '[' . While solving it, this one got missed. It is now addressed properly in all places by using a backslash '\' before the '<'.